pkg/asset/manifests/ingress.go

```diff
@@ -113,6 +118,7 @@ func (ing *Ingress) generateDefaultIngressController(config *types.InstallConfig
 						Scope: operatorv1.InternalLoadBalancer,
 					},
 				},
+				Replicas: replicas,
```

Comment:
I would have expected that the ingress operator would be deducing the correct number of replicas to use by inspecting the Infrastructure resource.